### PR TITLE
Show numbered title

### PR DESCRIPTION
**Pull request by goose-life**
closes https://github.com/laws-africa/indigo-lawsafrica/issues/150

On work:
image

On document:
image

image

Work popup:
image
image

Document popup:
image

### CODE REVIEW

**Comment by goose-life** [comment]
Our initial idea was to include it for the work only in the Overview, but I think it makes more sense to include it on all Work pages

File: indigo_app/templates/indigo_api/work_layout.html [hunk 1]
```diff
@@ -21,13 +21,14 @@
       {% block main-header-title %}
         {% if work.pk %}
           <h5 class="main-header-title">
-            <span class="work-title">{% if work.pk %}{{ work.title }}{% else %}Create a new work{% endif %}</span>
```
Comment (longhotsummer) [inline indigo_app/templates/indigo_api/work_layout.html:24]: I think this is necessary for the new work page?

File: indigo_app/templates/indigo_api/work_popup.html [hunk 2]
```diff
@@ -17,6 +17,10 @@ <h6>
     <br><span class="text-muted mt-1 wb-all">{{ work.frbr_uri }}</span>
   </h6>
 
+  {% if work.numbered_title %}
+    <p>{{ work.numbered_title }}</p>
```
Comment (longhotsummer) [inline indigo_app/templates/indigo_api/work_popup.html:21]:
```suggestion
    <p class="mb-2">{{ work.numbered_title }}</p>
```

File: indigo_app/templates/indigo_api/work_layout.html (hunk 1 above)
Comment (goose-life) [reply indigo_app/templates/indigo_api/work_layout.html:24]: the whole block is wrapped in `if work.pk`?

File: indigo/analysis/work_detail/base.py [hunk 3]
```diff
@@ -47,7 +47,7 @@ def work_friendly_type(self, work):
         return _('Act')
 
 
-class ChapterMixin:
+class ChapterWorkDetail(BaseWorkDetail):
```
Comment (longhotsummer) [inline indigo/analysis/work_detail/base.py:50]: What's the motivation behind this?

File: indigo_app/templates/indigo_api/work_layout.html (hunk 1 above)
Comment (goose-life) [reply indigo_app/templates/indigo_api/work_layout.html:24]: Currently I think it gives `(untitled work)` here for a new work

File: indigo/analysis/work_detail/base.py (hunk 3 above)
Comment (goose-life) [reply indigo/analysis/work_detail/base.py:50]: The ChapterMixin's `work_numbered_title` wasn't getting called at all. Chapters' numbered titles were being given as "Act x of yyyy"

File: indigo_app/templates/indigo_api/work_layout.html (hunk 1 above)
Comment (longhotsummer) [reply indigo_app/templates/indigo_api/work_layout.html:24]: Ah, you make a strong argument!

**Pull request merged**